src/MRA.Identity.Api/Controllers/EmailTemplatesController.cs

```diff
@@ -0,0 +1,49 @@
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using MRA.Identity.Application.Contract.EmailTemplates.Commands;
+using MRA.Identity.Application.Contract.EmailTemplates.Queries;
+
+namespace MRA.Identity.Api.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+[Authorize]
+public class EmailTemplatesController(ISender mediator)
+    : ControllerBase
+{
+    [HttpPost]
+    public async Task<IActionResult> Create([FromBody] CreateEmailTemplateCommand createCommand)
+    {
+        var slug = await mediator.Send(createCommand);
+        return Ok(slug);
+    }
+
+    [HttpDelete]
+    public async Task<IActionResult> Delete([FromQuery] string slug)
+    {
+        await mediator.Send(new DeleteEmailTemplateCommand { Slug = slug });
+        return Ok();
+    }
+
+    [HttpPut]
+    public async Task<IActionResult> Update([FromBody] UpdateEmailTemplateCommand updateCommand)
+    {
+        var slug = await mediator.Send(updateCommand);
+        return Ok(slug);
+    }
+
+    [HttpGet("getTemplate")]
+    public async Task<IActionResult> GetTemplate([FromQuery] string slug)
+    {
+        var emailTemplateResponse = await mediator.Send(new GetEmailTemplateQuery { Slug = slug });
+        return Ok(emailTemplateResponse);
+    }
+
+    [HttpGet("getSubjects")]
```

Comment:
Do we need this?

Reply:
yes because controller has two get methods

Reply:
Change [HttpGet("getSubjects")] to [HttpGet]. Route should be /emailtempates